homeassistant/components/fireservicerota/config_flow.py

```diff
@@ -0,0 +1,58 @@
+"""Config flow for FireServiceRota."""
+from pyfireservicerota import FireServiceRota, InvalidAuthError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_TOKEN, CONF_URL, CONF_USERNAME
+
+from .const import DOMAIN, URL_LIST  # pylint: disable=unused-import
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_URL, default="www.brandweerrooster.nl"): vol.In(URL_LIST),
```

Comment:
`vol.In` takes a dictionary with `value=>label`. Would it make more sense to show the name of the service, instead of the website url?

Reply:
Also, put the url first, as it will decide for the user what credentials to enter.

Reply:
Very good ideas, implemented!